EIPS/eip-165.md

```diff
@@ -171,7 +171,7 @@ contract ERC165Cache {
                                     30000,         // 30k gas
                                     _contract,     // To addr
                                     x,             // Inputs are stored at location x
-                                    0x8,           // Inputs are 8 bytes long
+                                    0x20,           // Inputs are 8 byes long
```

Comment:
Shouldn't this say "32 bytes"?

Reply:
Thank you. This is updated here https://github.com/fulldecent/EIPs/commit/734d68b6439941bf1577a4f1156b8e906185f7f5#diff-257877e0f2e79bcd3931e1058c6a6332

Reply:
I don't see the update. It still says `8 byes`.

Reply:
Sorry for lying! Please see the correction here https://github.com/fulldecent/EIPs/blame/patch-4/EIPS/eip-165.md#L155